### ISSUE TITLE
PyTorch missing?

### ISSUE THREAD

**Issue by ssrb19** [fimrie/DEVELOP#4]
Hello, while running the training command, I get:
 `No module named 'torch'
Failed to import torch`
Not sure how important it is, but do we need to install pytorch in the environment? If that is the case, perhaps a modification of the yml file would be helpful. Thanks!

**Comment by kate-fie** [comment]
Yes, you do not need to install pytorch in the env, and it's not included in the yml file. Thanks for clarifying!
